File: Competition/CompetitionsModel/src/main/java/competition/Registry.java
package competition;

public class Registry extends SystemUser {

    private boolean isAdmin;

    public Registry(String username, String password, String firstName, String lastName, boolean isAdmin) {
        super(username, password, firstName, lastName);
        this.isAdmin = isAdmin;
    }

    public boolean getIsAdmin() {
        return isAdmin;
    }

    public void setAdmin(boolean admin) {
        isAdmin = admin;
    }
}

File: CompetitionCSh/CompetitionCSh/domain/entities/Registry.cs
namespace CompetitionCSh.domain.entities;

public class Registry : SystemUser
{
    public Registry(string userName, string password, string firstName, string lastName, bool isAdmin) : base(
        userName, password, firstName, lastName)
    {
        this.isAdmin = isAdmin;
    }

    public bool isAdmin { get; set; }
}